inst/tests/froll.Rraw

```diff
@@ -468,25 +469,31 @@ ma = function(x, n, na.rm=FALSE, nf.rm=FALSE) {
 n = 4
 x = 1:16
 x[5] = NaN
-test(6000.120, frollmean(x, n), ma(x, n, nf.rm=TRUE))
+test(6000.120, frollmean(x, n), ma(x, n))
```

Comment:
We now don't need to remove non-finite because we handle them as expected. Clearly described breaking change in NEWS.